main.css

```diff
@@ -0,0 +1,162 @@
+:root {
+  /* Colores */
+  --background: #191765;
+  --primary-2: #7d73fc;
+  --primary-3: #9490ff;
+}
+
+body {
+  display: flex;
+  width: auto;
+  height: auto;
+  justify-content: center;
+  font-family: 'Roboto', sans-serif;
+  background-color: var(--background);
+}
+
+header {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header a {
+  padding-right: 15px;
+}
+
+main {
+  background-color: #191765;
+  background-image: url(/assets/img/TwoCircles.png);
```

Comment:
I suggest you check the URL path properly as this is what it looks like: [Live demo](https://rawcdn.githack.com/alef-garrido/PortfolioAlef/1073795fdbe580c825b24e0b0ed07572125f00e4/index.html#)